klippa/src/serialize.rs

```diff
@@ -0,0 +1,394 @@
+//! serializer
```

Comment:
assuming this is based on harfbuzz, it would be useful to link to the particular source that we're referencing, both here and for particular items below.